minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java

```diff
@@ -72,27 +79,35 @@ public static LocalAlluxioMaster create() throws IOException {
     String workDirectory = uniquePath();
     FileUtils.deletePathRecursively(workDirectory);
     ServerConfiguration.set(PropertyKey.WORK_DIR, workDirectory);
-    return create(workDirectory);
+    return create(workDirectory, null, null);
   }
 
   /**
    * Creates a new local Alluxio master with a isolated port.
    *
    * @param workDirectory Alluxio work directory, this method will create it if it doesn't exist yet
+   * @param rpcBindSocket the socket whose address the master's RPC server will bind to
+   * @param webBindSocket the socket whose address the master's web server will bind to
    * @return the created Alluxio master
    */
-  public static LocalAlluxioMaster create(final String workDirectory) throws IOException {
+  public static LocalAlluxioMaster create(final String workDirectory,
+      @Nullable ServerSocket rpcBindSocket,
+      @Nullable ServerSocket webBindSocket) throws IOException {
```

Comment:
we can avoid the `@Nullable` parameters with another method overload

Reply:
sg